src/openklant/components/klantinteracties/migrations/0037_digitaaladres_verificatie_datum.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.2.1 on 2025-07-30 14:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("klantinteracties", "0036_alter_partij_voorkeurs_digitaal_adres_and_more"),
+    ]
+
+    operations = [
```

Comment:
Alex also mentioned in the issue that it should be possible to verify DigitaalAdressen in bulk (because by default existing adressen won't be verified), so I think it would be good to have a management command to do this

Reply:
added command `bulk_verify_digitaal_adressen`